store/src/java/com/zimbra/cs/service/mail/ToXML.java

```diff
@@ -2959,6 +2979,13 @@ private static void addContent(Element elt, MPartInfo mpi, int maxSize, boolean
                     data = data.substring(0, maxSize);
                 }
             }
+            if(warningMsg.length()>0){
```

Comment:
You can club both conditions in a single if

Reply:
clubbed both the conditions now